### PR TITLE
Feature: Added organization auth

### PR DESCRIPTION
**Pull request by subru-37**

### CODE REVIEW

File: src/Components/FormSample/FormSample.jsx
Comment (jaison080) [inline src/Components/FormSample/FormSample.jsx:1]: Change the filename from FormSample to something which it is specific for or else debugging would be difficult for others.

File: src/Components/InviteToProjectModal/InviteToProjectModal.js
Comment (jaison080) [inline src/Components/InviteToProjectModal/InviteToProjectModal.js:1]: Here only the mail function needs to be commented right.The invite functionality should work fine right?

File: src/Components/JoinTeamModal/JoinTeamModal.js
Comment (jaison080) [inline src/Components/JoinTeamModal/JoinTeamModal.js:1]: Here only the mail function needs to be commented right.The request functionality should work fine right?

File: src/Components/Request/Received.js
Comment (jaison080) [inline src/Components/Request/Received.js:1]: In these also the mail function only needs to be commented out right rest all are firebase functions right,

**Review by jaison080 — CHANGES_REQUESTED** [review]

File: src/Components/Request/Received.js
Comment (subru-37) [reply src/Components/Request/Received.js:1]: Yes, I will comment out the .then() part which handles the email function

**Review by jaison080 — APPROVED** [review]

**Pull request merged**